### ISSUE TITLE
2.0.1 failing to npm install

### ISSUE THREAD

**Issue by fantapop** [mileszim/webpurify#19]
I'm not entirely sure whats going on here.  I just tried to update to 2.0.1 and I'm unable to yarn install or npm install with the same error.  Without getting too deep it looks like the postinstall script is running the build after its npm installed . The build relies on babel-cli which is listed only in devDependencies.

```
sh: babel: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! webpurify@2.0.1 build: `babel src --out-dir dist`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the webpurify@2.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/chris/.npm/_logs/2018-05-04T07_40_22_009Z-debug.log
npm WARN asset-api@0.0.1 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! webpurify@2.0.1 postinstall: `npm run-script build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the webpurify@2.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/chris/.npm/_logs/2018-05-04T07_40_23_139Z-debug.log
```

And here is yarns version, its very similar:

```
error An unexpected error occurred: "/Users/chris/Development/asset-api/node_modules/webpurify: Command failed.
Exit code: 1
Command: sh
Arguments: -c npm run-script build
Directory: /Users/chris/Development/asset-api/node_modules/webpurify
Output:
> webpurify@2.0.1 build /Users/chris/Development/asset-api/node_modules/webpurify
> babel src --out-dir dist

sh: babel: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! webpurify@2.0.1 build: `babel src --out-dir dist`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Fanled at thn wnbpurify@2.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?
```

When I go in to the webpurify directory after install and try to run yarn build I get this output:

```
$ yarn build
yarn run v1.5.1
$ ./node_modules/.bin/babel src --out-dir dist
sh: ./node_modules/.bin/babel: No such file or directory
error An unexpected error occurred: "Command failed.
Exit code: 127
Command: sh
Arguments: -c ./node_modules/.bin/babel src --out-dir dist
Directory: /Users/chris/Development/asset-api/node_modules/webpurify
Output:
".
```

**Comment by mileszim** [comment]
Ah nuts, that's no good. My thinking here is utilizing the post install npm hook to transpile on install to the node version installing it, ensuring the built-in es6 code is preferred when it can. I'm not sure of another way to approach it besides setting build targets for each major version of node and including a build file for each one, deciding which is included at runtime based on the process node version. Any other ideas @fantapop ?

**Comment by fantapop** [comment]
This was just on a default install.

 One approach for what you want to accomplish is to have an npm script that
will build and push your transpiled source up to npm. You can set up your
package.json to point to the cjs and the esm source so that people using
your module will choose the appropriate build.

**Comment by mileszim** [comment]
This should be fixed now with release 2.1.0
